myapp/spec/system/tasks_spec.rb

```diff
@@ -3,7 +3,10 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
-  let(:submit_button_text) { I18n.t('helpers.submit.submit', model: I18n.t('activerecord.models.task')).capitalize }
+  context 'when user logged in' do
+    let(:user) { create(:user) }
+    let(:rspec_session) { {user_id: user.id} }
+    let(:submit_button_text) { I18n.t('helpers.submit.submit', model: I18n.t('activerecord.models.task')).capitalize }
 
     describe '#index' do
       it 'displays a create task link' do
```

Comment:
I think we need a test to ensure displays tasks created by logged in user

Reply:
make sense~